style.css

```diff
@@ -274,6 +278,11 @@ button[type="submit"] {
   text-align: center;
 }
 
+.error-message {
+  color: #c8102e;
+  font-size: 0.7rem;
```

Comment:
If you make this `position: absolute` it will fix the problem with your labels

Reply:
Then, to fix the checkbox error text, I wrapped the `.checkbox-field` in a `.form-field` div and updated the javascript accordingly.

```
<div class="form-field">
          <div class="checkbox-field" id="privacy">
            <input
              type="checkbox"
              name="terms-of-service"
              id="terms-of-service"
              required
            />
            <label for="terms-of-service">
              I agree to the following:
              <span class="red"
                ><a href="#">Privacy Policy,</a><br />
                <a href="#">Tims Rewards Terms and Conditions,</a>
                <a href="#">Terms of Service</a></span
              >
            </label>
          </div>
          <div class="error-message"></div>
        </div>
```

```
 // We use && to say "and"
      if (input.type === "checkbox" && input.id === "terms-of-service") {
        input
          .closest(".form-field")
          .querySelector(".error-message").textContent =
          "You must agree to the privacy policy and terms of service before signing up.";
      }
```

and added the following css to undo position absolute on this part (it pushes the button down on the real site so it's not absolute here)

```
.checkbox-field ~ .error-message {
  position: static;
}
```

Could still use a little tweaking but you get the idea

<img width="468" alt="Screen Shot 2021-02-08 at 7 12 11 PM" src="https://user-images.githubusercontent.com/8116130/107297538-962ecc80-6a41-11eb-9e4a-c292177bffb6.png">